src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -542,10 +545,27 @@ private[cassandra] object CassandraJournal {
 }
 
 class FixedRetryPolicy(number: Int) extends RetryPolicy {
-  override def onUnavailable(statement: Statement, cl: ConsistencyLevel, requiredReplica: Int, aliveReplica: Int, nbRetry: Int): RetryDecision = retry(cl, nbRetry)
-  override def onWriteTimeout(statement: Statement, cl: ConsistencyLevel, writeType: WriteType, requiredAcks: Int, receivedAcks: Int, nbRetry: Int): RetryDecision = retry(cl, nbRetry)
-  override def onReadTimeout(statement: Statement, cl: ConsistencyLevel, requiredResponses: Int, receivedResponses: Int, dataRetrieved: Boolean, nbRetry: Int): RetryDecision = retry(cl, nbRetry)
-  override def onRequestError(statement: Statement, cl: ConsistencyLevel, cause: DriverException, nbRetry: Int): RetryDecision = tryNextHost(cl, nbRetry)
+  override def onUnavailable(statement: Statement, cl: ConsistencyLevel, requiredReplica: Int, aliveReplica: Int,
+                             nbRetry: Int): RetryDecision = {
+    if (nbRetry == 0)
+      tryNextHost(cl, nbRetry) // see DefaultRetryPolicy
+    else
+      retry(cl, nbRetry)
```

Comment:
So, go to next host if this is the first failure, but then stay with the next host? What if the next host also fails?